scripts/core/editor2/editor.js

```diff
@@ -698,7 +698,8 @@ angular.module('superdesk.apps.editor2', [
     'superdesk.apps.editor2.utils',
     'superdesk.apps.spellcheck',
     'superdesk.apps.authoring',
-    'angular-embed'
+    'angular-embed',
+    'angular-embed.handlers'
```

Comment:
it seems to be the other way around https://github.com/superdesk/angular-embed/blob/master/dist/angular-embed.js#L28